app/api/issues/edit/[id]/route.ts

```diff
@@ -0,0 +1,25 @@
+import { editIssueSchema } from '@/features/issue/schemas'
+import { NextRequest, NextResponse } from 'next/server'
+import prisma from '@/prisma/db'
+import { z } from 'zod'
+import { Status } from '@prisma/client'
+
+export const PATCH = async (request: NextRequest) => {
+	const id = parseInt(request.url.split('/').slice(-1)[0])
```

Comment:
export const PATCH = async (request: NextRequest, {params}: {params:{id:number}}) => {

const id = params.id
}

coś takiego chyba zadziała, jak nie to popatrz tu w docsach, bo const id = parseInt(request.url.split('/').slice(-1)[0]) brzydkie jest i tak sie nie robi 
 https://nextjs.org/docs/app/building-your-application/routing/route-handlers#extended-nextrequest-and-nextresponse-apis